gameart.md

```diff
@@ -0,0 +1,57 @@
+# Board in the start menu
+====================================================================================================
+
+
+
+
+
+                                   Press ENTER to start the game
+                                   Press I for the informations menu
+
+
+
+
+
+====================================================================================================
+Any other key entry will cause you to quit the game...
+====================================================================================================
+
+
+
+# Board during the game
+====================================================================================================
+
+
+
+
+|                                                                                                  |
+|                                                o                                                 |
+|                                                                                                  |
+|                                                                                                  |
+
+
+
+
+====================================================================================================
+Score : X - X                                                           Longest rally : X (YY/DD/MM)
+====================================================================================================
+
+
+
+# Board in the informations menu
+====================================================================================================
+
+
+
+                            The single gamemode is a 1v1 between two players.
+                            Your goal ? Score 3 points before your opponent.
+
+                            Controls : 
+                                Left-player	 --> Q to move up and A to move down
```

Comment:
Agree to using Q-A and O-L for control, could you help to revise the README.md to align?

Reply:
Of course, I fixed both problems